README.md

```diff
@@ -50,6 +50,10 @@ After merge to staging, the code quality is everyone’s responsibility.
 
 ## Site Architecture
 
+| ![AIID project arquitecture](https://user-images.githubusercontent.com/6564809/181660368-990b54ea-503d-4cec-bd81-ca134f5e9ff2.jpeg) |
```

Comment:
Nice! The JPEG compression makes the diagram look blurry, though. I think it should be a PNG or SVG if possible.

Reply:
@lmcnulty I changed to PNG and it looks better. Maybe a little bit blurry on some components but I think it's because of the README width limitation.